app/models/validations/sales/financial_validations.rb

```diff
@@ -85,6 +85,22 @@ def validate_percentage_owned_not_too_much_if_older_person(record)
     end
   end
 
+  def validate_equity_in_range_for_year_and_type(record)
+    return unless record.type && record.equity && record.collection_start_year
+
+    ranges = EQUITY_RANGES_BY_YEAR.fetch(record.collection_start_year, DEFAULT_EQUITY_RANGES)
+
+    if (range = ranges[record.type])
```

Comment:
nested ifs
could reduce nesting by changing 93 to return unless (range = ranges[record.type])